src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -10023,6 +10023,7 @@ public static partial class RandomAccess
         public static void Write(Microsoft.Win32.SafeHandles.SafeFileHandle handle, System.ReadOnlySpan<byte> buffer, long fileOffset) { }
         public static System.Threading.Tasks.ValueTask WriteAsync(Microsoft.Win32.SafeHandles.SafeFileHandle handle, System.Collections.Generic.IReadOnlyList<System.ReadOnlyMemory<byte>> buffers, long fileOffset, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public static System.Threading.Tasks.ValueTask WriteAsync(Microsoft.Win32.SafeHandles.SafeFileHandle handle, System.ReadOnlyMemory<byte> buffer, long fileOffset, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static void FlushToDisk(Microsoft.Win32.SafeHandles.SafeFileHandle handle) { throw null; }
```

Comment:
nit - please put in alpha order. we have tools that regenerate this and it will otherwise cause a diff

Reply:
OK...before I do, I notice that `SetLength` appears before the various `Read` and `ReadAsync` methods, when it should appear after them in order to preserve alpha order. Should I make that change in this PR too or is it best to use a separate issue?

Reply:
If you run the tool it will take care of it

https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md